packages/queries/src/schema/getTableSchema.ts

```
@@ -0,0 +1,56 @@
+    const columns = getColumnNamesAndDefs(tableDef)
+        // Only include selectable columns
+        .filter(([_, columnDef]) => {
+            return isSelectableColumn(columnDef);
+        })
+        .map(([columnName, columnDef]) => {
+            return {
+                [columnName]: convertToTypeBoxSchema(columnDef.properties.type),
+            };
+        })
+        .reduce((acc, curr) => {
+            return { ...acc, ...curr };
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)